tests build failed on mac

Hi community:

I've installed Catch2, but the tests build failed (tried `make` or `cmake --build build --verbose`):
```
Consolidate compiler generated dependencies of target openshot_protobuf
[  5%] Built target openshot_protobuf
[  6%] Automatic MOC for target openshot
[  6%] Built target openshot_autogen
Consolidate compiler generated dependencies of target openshot
[ 62%] Built target openshot
[ 62%] Automatic MOC for target openshot-player
[ 62%] Built target openshot-player_autogen
Consolidate compiler generated dependencies of target openshot-player
[ 64%] Built target openshot-player
Consolidate compiler generated dependencies of target openshot-html-example
[ 66%] Built target openshot-html-example
Consolidate compiler generated dependencies of target openshot-example
[ 67%] Built target openshot-example
Consolidate compiler generated dependencies of target pyopenshot
[ 70%] Built target pyopenshot
[ 71%] Building CXX object tests/CMakeFiles/catch-main.dir/catch_main.cpp.o
/Users/ted/Documents/GitHub/libopenshot_repo/libopenshot/tests/catch_main.cpp:32:10: fatal error: 'catch2/catch.hpp' file not found
#include <catch2/catch.hpp>
         ^~~~~~~~~~~~~~~~~~
1 error generated.
make[2]: *** [tests/CMakeFiles/catch-main.dir/catch_main.cpp.o] Error 1
make[1]: *** [tests/CMakeFiles/catch-main.dir/all] Error 2
make: *** [all] Error 2
```

Config command is here:
```
cmake -G "Unix Makefiles" -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_C_COMPILER=clang -DCMAKE_PREFIX_PATH=$(brew --prefix qt5) -DCMAKE_EXE_LINKER_FLAGS="-stdlib=libc++" -DCMAKE_SHARED_LINKER_FLAGS="-stdlib=libc++" -D"CMAKE_INSTALL_RPATH_USE_LINK_PATH=1" -D"ENABLE_RUBY=0" -D"CMAKE_BUILD_TYPE:STRING=Release" -D"ENABLE_TESTS=1"  ../
```

Result:
```
-----------------------------------------------------------------
          Welcome to the OpenShot Build System!

CMake will now check libopenshot's build dependencies and inform
you of any missing files or other issues.

For more information, please visit <http://www.openshot.org/>.
-----------------------------------------------------------------

Generating build files for OpenShot with CMake 3.21.0
  Building libopenshot (version 0.2.5)
  SO/API/ABI Version: 19

-- Could NOT find ImageMagick (missing: ImageMagick_Magick++_LIBRARY ImageMagick_MagickCore_LIBRARY)
-- Looking for system jsoncpp
-- Could NOT find Resvg, using Qt SVG parsing instead (missing: Resvg_LIBRARIES Resvg_INCLUDE_DIRS)
CMake Deprecation Warning at bindings/python/CMakeLists.txt:34 (cmake_policy):
  The OLD behavior for policy CMP0078 will be removed from a future version
  of CMake.

  The cmake-policies(7) manual explains that the OLD behaviors of all
  policies are deprecated and that a policy should be set to OLD only under
  specific short-term circumstances.  Projects should be ported to the NEW
  behavior and not rely on setting a policy to OLD.


-- PYTHON_MODULE_PATH: /usr/local/python
-- Doxygen found, documentation target enabled
-- Build configuration:
-- The following features have been enabled:

 * FFmpeg swresample, Audio resampling uses swresample
 * FFmpeg hwaccel, GPU-accelerated routines (FFmpeg 3.4+)
 * OpenCV algorithms, Use OpenCV algorithms
 * Parallel tests, Unit tests can use 12 processors
 * Unit tests, Compile unit tests for library functions
 * Non-default target 'coverage', Run unit tests and (if enabled) collect coverage data
 * Non-default target 'doc', Build formatted API documentation (HTML+SVG)

-- The following OPTIONAL packages have been found:

 * PkgConfig
 * OpenCV (required version >= 4)
 * Threads
 * PythonInterp (required version >= 3)
 * PythonLibs (required version >= 3)
 * Doxygen
 * Catch2

-- The following REQUIRED packages have been found:

 * OpenShotAudio (required version >= 0.2.0)
 * jsoncpp
 * Qt5Widgets
 * FFmpeg
 * OpenMP
 * ZeroMQ
 * Protobuf (required version >= 3)
 * Qt5Core (required version >= 5.15.2)
 * Qt5Gui
 * Qt5
 * SWIG (required version >= 3.0)

-- The following features have been disabled:

 * Coverage, analyze test coverage and generate report
 * IWYU (include-what-you-use), Scan all source files with 'iwyu'

-- The following OPTIONAL packages have not been found:

 * ImageMagick
 * cppzmq
 * Resvg

-- Configuring done
-- Generating done
-- Build files have been written to: /Users/ted/Documents/GitHub/libopenshot_repo/libopenshot/build
```

And Catch2 is surely installed:
```
$ brew --prefix catch2
/usr/local/opt/catch2
```

How to solve this? Thanks.

--- a/tests/CMakeLists.txt
+++ b/tests/CMakeLists.txt
@@ -95,6 +95,7 @@ endif()
 # Create object library for test executable main(),
 # to avoid recompiling for every test
 add_library(catch-main OBJECT catch_main.cpp)
+target_link_libraries(catch-main PUBLIC Catch2::Catch2)
 
 foreach(tname ${OPENSHOT_TESTS})
   add_executable(openshot-${tname}-test ${tname}.cpp $<TARGET_OBJECTS:catch-main>)

